crates/bevy_math/src/face_toward.rs

```diff
@@ -1,8 +1,51 @@
 use crate::{Mat4, Vec3};
 
-/// Generates a translation / rotation matrix that faces a given target
+/// Implements a function that generates a translation / rotation matrix that faces a given target.
+///
+/// # Example
+///
+/// ```rust
+/// # use bevy_math::{FaceToward, Vec3, Vec4};
+/// # #[allow(dead_code)]
```

Comment:
is the `#[allow(dead_code)]` still needed? it seems it's used below

Reply:
It was throwing warnings when I was testing the example in a separate crate. Since all the bevy_math modules are likely to be deleted/moved this documentation will be gone as well. I'll mark this as resolved and get started with cleaning up the bevy_math crate.